packages/document-service-api/src/routes/v2/back-office/access-rules.js

```diff
@@ -3,40 +3,55 @@ const {
 	LPA_USER_ROLE,
 	VIRUS_CHECK_STATUSES
 } = require('@pins/common/src/constants');
-const documentTypes = require('@pins/common/src/document-types');
+const { getDocType } = require('@pins/common/src/document-types');
+
+const CLIENT_CREDS_ROLE = 'client-credentials';
+module.exports.CLIENT_CREDS_ROLE = CLIENT_CREDS_ROLE;
 
 /**
+ * @typedef {import('@pins/common/src/document-types').DocType} DocType
  * @typedef {import('appeals-service-api/index').Schema.Document} DocMetaData
- * @typedef {function(DocMetaData, {owner: string}): boolean} PermissionsCheck
+ * @typedef {function(DocMetaData, DocType): boolean} PermissionsCheck
+ */
+
+/**
+ * @type {PermissionsCheck}
+ */
+const publicDocAccess = (docMetaData, docType) => {
+	if (!docMetaData.published || !docMetaData.redacted) return false;
+	if (docType.publiclyAccessible) return true; // todo: set these to true when we know which data model docs appear on public urls
+	return false;
+};
```

Comment:
```suggestion
const publicDocAccess = (docMetaData, docType) => 
	!!docMetaData.published && !!docMetaData.redacted &&  docType.publiclyAccessible;
```

Reply:
Not sure if published and redacted are bools but if they are obvi no !!

Reply:
yep have made this change, think I always fail to see this sort of thing after I've been playing around for a while